### PR TITLE
Use native R testing

### PR DESCRIPTION
**Pull request by krlmlr**
for Travis-CI.

### CODE REVIEW

**Comment by hadley** [comment]
Can we switch to modern R support?

```yaml
language: r
cache: packages
```

**Comment by krlmlr** [comment]
Seems to work well.

File: .travis.yml [hunk 1]
```diff
@@ -1,22 +1,7 @@
...
-  - ./travis-tool.sh install_deps
-
-script: ./travis-tool.sh run_tests
-
-after_failure:
-  - ./travis-tool.sh dump_logs
-
-env:
-  - WARNINGS_ARE_ERRORS=1
+warnings_are_errors: true
```
Comment (hadley) [inline .travis.yml]: You don't need that any more - it's the default

**Comment by craigcitro** [comment]
LGTM (feel free to merge after dropping `warnings_are_errors`)

**Pull request merged**